tutorials/slack_integration.md

```diff
@@ -0,0 +1,193 @@
+<a name="slack-events"></a>
+# Integrations - Slack Events
+
+## Table of Contents
+
+* [Credential Storage](#credentials)
+* [Creating endpoint](#endpoint)
+* [Slack Setup](#setup)
+* [Setting Up a Webserver](#webserver)
+* [Conclusion](#conclusion)
+
+### Overview
+
+This tutorial will go over how to setup a simple integration with Slack that allows us to send email using a slack slash command. You'll need the following to complete this tutorial.
+
+* SendGrid API Key
+* Slack OAuth Token
+* Slack account & Valid Slack Workspace
+* ngrok (if you do not have an publicly accessible web endpoint)
+* Sinatra 
+* Sendgrid ruby gem
+
+Slack allows us to create [apps](https://api.slack.com/slack-apps). Apps can have different types of capabilities. One such capability is a [slash command](https://api.slack.com/slash-commands). Slash Commands enable Slack users to interact with your app directly from Slack. We will write a simple slash command with following structure.
```

Comment:
This link is really small and short:

> create [apps]

Instead, to increase link usability, let's expand the linked text.

    [create apps]